src/main/java/uk/gov/ons/census/fwmt/jobservice/rabbit/TransientExceptionHandler.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.ons.census.fwmt.jobservice.rabbit;
+
```

Comment:
This class does not have my full changes as I changed MAX_RETRY_COUNT to be a configurable value and some other changes.